### PR TITLE
Add MilkyWay

### PR DESCRIPTION
**Pull request by RiccardoM**
Adds `milkyway` chain information

### CODE REVIEW

File: cosmos/milkyway.json [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/milkyway/chain.png",
+  "nodeProvider": {
+    "name": "MilkyWay Labs",
+    "email": "development@milkyway.zone"
+  },
+  "stakeCurrency": {
+    "coinDenom": "MILK",
+    "coinMinimalDenom": "umilk",
+    "coinDecimals": 6,
+    "coinGeckoId": "",
```
Comment (editaahn) [inline cosmos/milkyway.json:15]: Please remove it if there's no available CoinGecko id.

File: cosmos/milkyway.json [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/milkyway/chain.png"
+  },
+  "walletUrl": "https://beta.milkyway.zone/",
+  "walletUrlForStaking": "",
```
Comment (editaahn) [inline cosmos/milkyway.json:19]: Remove these two lines. walletUrl is deprecated and walletUrlForStaking shouldn't be empty.

File: cosmos/milkyway.json [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+      "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/osmosis/factory/osmo1f5vfcph2dvfeqcqkhetwv75fda69z7e5c2dldm3kvgj23crkv6wqcn47a0/milktia.png",
+      "gasPriceStep": {
+        "low": 0.0025,
+        "average": 0.003,
+        "high": 0.004
+      }
+    },
+    {
+      "coinDenom": "stTIA",
+      "coinMinimalDenom": "umilk",
```
Comment (editaahn) [inline cosmos/milkyway.json:100]: The denom and the minimal denom don't seem to match

File: cosmos/milkyway.json [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+      "coinDecimals": 6,
+      "coinGeckoId": "",
```
Comment (editaahn) [inline cosmos/milkyway.json:102]: Remove this line as well.

File: cosmos/milkyway.json [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+      "coinImageUrl": "ibc/8D4FC51F696E03711B9B37A5787FB89BD2DDBAF788813478B002D552A12F9157",
```
Comment (editaahn) [inline cosmos/milkyway.json:103]: Please provide a url.

File: images/milkyway/chain.png
Comment (editaahn) [inline images/milkyway/chain.png:1]: Image sizes should be 256*256px.

File: cosmos/milkyway.json [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+      "coinMinimalDenom": "ibc/6C349F0EB135C5FA99301758F35B87DB88403D690E5E314AB080401FEE4066E5",
+      "coinDecimals": 6,
+      "coinGeckoId": "celestia",
+      "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/celestia/chain.png",
+      "gasPriceStep": {
+        "low": 0.0025,
+        "average": 0.003,
+        "high": 0.004
+      }
+    },
```
Comment (editaahn) [inline cosmos/milkyway.json:85]: duplicated

**Review by editaahn — COMMENTED** [review]
Hi! Could you please rewrite the information based on the README? You can test the code in your local environment and see how to do it on the README.

File: cosmos/milkyway.json (hunk 6 above)
Comment (RiccardoM) [reply cosmos/milkyway.json]: This is actually not duplicated, we simply have two versions of TIA supported inside our chain: one that is bridged through Osmosis, and the other ones that is bridged through Celestia directly. We support both as our fee tokens.

**Pull request merged**